scripts/check_cmake_files.py

```diff
@@ -23,30 +31,43 @@
 import subprocess
 import sys
 
+from pprint import pprint as pp
 
-DEFAULT_DIRS = [
+
+# Directories relative to the repo root that will be scanned by default if no
+# arguments are passed
+_DEFAULT_DIRS = [
     'Firestore/core',
     'Firestore/Source'
 ]
 
+# When scanning the filesystem look for specific files or files with these
```

Comment:
Ultranit: missing comma in-between `filesystem` and `look`.

Reply:
Done.